4_predictions.yml

```diff
@@ -11,22 +11,33 @@ sources:
 targets:
   4_predictions:
     depends:
-      - out_data/reservoir_io_sntemp.csv
-      - out_data/reservoir_downstream_preds.csv
+      - out_data/reservoir_outlet_depth_preds.csv
       - out_data/dwallin_stream_preds.csv.zip
+      - out_data/forecast_pgdl_da_outputs.zip
 
-  # file from delaware-model-prep/9_collaborator_data/res/
-  out_data/reservoir_io_sntemp.csv:
-    command: copy_filter_feather(
-      out_csv = target_name,
-      in_feather = 'in_data/res_io_sntemp.feather',
-      site_ids = reservoir_modeling_site_ids)
+  # Reservoir outlet temperatures from GLM (files from delaware-model-prep/9_collaborator_data/res/)
+  out_data/reservoir_outlet_depth_preds.csv:
+    file.copy(from = 'in_data/outlet_depth_preds.csv', to = target_name)
+  
+  # out_data/reservoir_io_sntemp.csv:
+  #   command: copy_filter_feather(
+  #     out_csv = target_name,
+  #     in_feather = 'in_data/res_io_sntemp.feather',
+  #     site_ids = reservoir_modeling_site_ids)
       
   out_data/reservoir_downstream_preds.csv:
     command: file.copy(from = 'in_data/downstream_preds.csv', to = target_name)
   
+  # DWALLIN
   out_data/dwallin_stream_preds.csv.zip:
     command: zip_obs(out_file = target_name, in_file = 'in_data/dwallin_stream_preds.csv')
     
-  #out_data/forecast[2021-04-16_2021-07-16]_files.zip:
-   # command: file.copy(from = 'in_data/forecast[2021-04-16_2021-07-16]_files.zip', to = target_name)
\ No newline at end of file
+  # PGDL-DA 
+  out_data/forecast_pgdl_da_outputs.zip:
+    command: utils::zip(files = 'in_data/PGDL-DA', zip = target_name)
+    
+  # Paired predictions  
+  out_data/all_mods_with_obs.rds:
+    command: file.copy(from = 'in_data/all_mods_with_obs.rds', to = target_name)
```

Comment:
do we want to convert this from `.rds` to `.csv` ?

Reply:
Yes -- `.rds` isn't very user-friendly for non-R users, so we convert all `.rds` to `.csv`.